doc/tutorials/klondike/app/README.md

```diff
@@ -0,0 +1,16 @@
+# app
```

Comment:
This is the default readme, either modify it to explain the example, or remove it.

Reply:
Flutter-generated file. We need to eradicate commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 which unintentionally added and committed it.

Reply:
The offending commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 has been reverted.